src/main/java/org/embulk/filter/expand_json/ExpandJsonFilterPlugin.java

```diff
@@ -38,11 +38,13 @@
         @Config("expanded_columns")
         public List<ColumnConfig> getExpandedColumns();
 
-        // Time zone of timestamp columns if the value itself doesn’t include time zone description (eg. Asia/Tokyo)
         @Config("time_zone")
         @ConfigDefault("\"UTC\"")
+        @Deprecated
```

Comment:
This annotation is for developers. It's better to throw `ConfigException`(or removing) for users, I think. It's because, after updated to this version, some users cannot know `time_zone` does not work if not throwing, then even this could cause an analysis using incorrect data and making important business decisions. This should be avoided.